examples/simple_osmesa.c

```diff
@@ -0,0 +1,209 @@
+//========================================================================
```

Comment:
I'll merge this file as `offscreen.c` if it is built for all platforms, uses `stb_image_write.h` and sets `GLFW_VISIBLE` to false.